src/main/java/com/dataguise/saas/dto/SourcesDetails.java

```diff
@@ -0,0 +1,97 @@
+package com.dataguise.saas.dto;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+public class SourcesDetails {
+    private int sourceId;
+    private String sourceName;
+    private String sourceType;
+    private String sourceModule;
+    private String sourceAgent;
+    private String sourceHostName;
+    private Integer sourcePortNumber;
+    private List<String> sourceDatabases;
+    private List<String> sourceSchemas;
+
+    public Timestamp getCreatedAt() {
+        return createdAt;
+    }
+
+    public void setCreatedAt(Timestamp createdAt) {
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.design.DesignForExtensionCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/SourcesDetails.java L21
    
    Class &#39;SourcesDetails&#39; looks like designed for extension (can be subclassed), but the method &#39;setCreatedAt&#39; does not have javadoc that explains how to do that safely. If class is not designed for extension consider making the class &#39;SourcesDetails&#39; final or making the method &#39;setCreatedAt&#39; static/final/abstract/empty, or adding allowed annotation for the method.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-157698824>*